apps/right-to-rent-check/acceptance/features/privacy-policy.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('Given I am on /privacy-policy');
+
+Before((
+  I,
+  privacyPolicyPage
+) => {
+  I.visitPage(privacyPolicyPage, steps);
```

Comment:
Is it ok to use visitPage in this step rather than completeToStep?

Reply:
I personally don't see any point in functional testing static pages.

Reply:
I agree, I think it just adds more time to run tests and more time to maintain.  I'll remove it.

Reply:
I've now updated